.github/workflows/publish.yml

```diff
@@ -0,0 +1,26 @@
+name: Publish
+on:
+  push:
+    tags:
+      - "v0.*"
```

Comment:
nit: It looks like you forgot to update after `1.0` 😄 (Just comment)

Reply:
Yeah, I'll take care of it! 😃